website/pages/docs/plugins/sources/aws/configuration.mdx

```diff
@@ -263,6 +268,28 @@ This is used to specify one or more accounts to extract information from. Note t
 
   List of OU member accounts to skip. This is useful in conjunction with `organization_units` if there are accounts under the selected OUs that should be ignored.
 
+### event_based_sync
```

Comment:
```suggestion
### event_based_sync

Event-based syncing is only supported by the enterprise-version AWS plugin. [Read more about event-based syncs]([/docs/plugins/sources/aws/event-based-sync](https://github.com/cloudquery/cloudquery/pull/13357/files#diff-d6b3e143b73c78cee7e26fed26091b5d29df661282616ba5ba58f010dadee2be)).
```

Reply:
(I think we should just re-iterate that this is enterprise-only, in case users didn't see it in the section higher up)